src/usb.ts

```diff
@@ -199,15 +199,23 @@ export class USB extends EventDispatcher {
                 }
 
                 function selectFn(device: USBDevice) {
+                    if (!(device instanceof USBDevice)) {
```

Comment:
I'm not sure about this addition. It assumes all devices the user will interact with will explicitly inherit from `USBDevice`. In reality it could easily be an object that looks like USBDevice and should still work.

Recommend this guard is removed (or replaced with something that checks for the properties we need).

Reply:
Ideally `getDevices` will use local information of available devices and list them from the local cache. If something different is added then returning that information will contradict the API.